gradle.properties

```diff
@@ -34,7 +34,7 @@ projectLicenseDistribution=repo
 projectIssueSystem=JIRA
 projectIssueUrl=https://issues.jasig.org/browse/UP
 
-gradleVersion=3.0
+gradleVersion=3.5
```

Comment:
@stalele the gradle reversion was dropped back to 3.4 in https://github.com/Jasig/uPortal/commit/fbafedf445408bdc91abbaf919c32d2a66398dc9